### PR TITLE
Update modal backdrop

### PR DESCRIPTION
**Pull request by prestonp**

### CODE REVIEW

File: packages/cf-style-const/src/variables.js [hunk 1]
```diff
@@ -97,5 +97,5 @@ export default {
   colorFacebookBlue: '#3b5998',
   colorMainBackground: '#e6e6e6',
   zIndexMax: 1000,
-  modalBoxShadow: '0 0 0 9999px rgba(0, 0, 0, .7)'
+  modalBackdropBackground: 'rgba(0, 0, 0, .7)'
```
Comment (prestonp) [inline packages/cf-style-const/src/variables.js]: Hey no worries! This is for a backdrop that covers the entire page. So it's a completely different pattern

Applied to a gh comment
```css
box-shadow: 0 1px 15px rgba(0,0,0,.75);
```

Cover entire page
screen shot 2017-06-06 at 2 44 28 pm

Comment (jwineman) [inline packages/cf-style-const/src/variables.js]: We probably need `rgba()` in a color const in `cf-style-const` too right?

Comment (adamschwartz) [inline packages/cf-style-const/src/variables.js]: You guys use `background: #000; opacity: .7` for your modal backdrop. So perhaps you’ll want to copy those two constants from there.

Comment (manatarms) [inline packages/cf-style-const/src/variables.js]: Hmm. I think there might be a difference between our react and bb modals.

Comment (prestonp) [inline packages/cf-style-const/src/variables.js]: @jwineman Not sure I understand, you want `rgba(0,0,0,0.7)` as a generic color const that's unrelated to modals?

Maybe something like overlayColor: 'rgba(0,0,0,0.7)` ?

Comment (adamschwartz) [inline packages/cf-style-const/src/variables.js]: > Maybe something like overlayColor: 'rgba(0,0,0,0.7)' ?

That would be very reasonable to me. If you want this to be in sync with the existing modals in the CF Dash (not sure whether those are "react" or "bb" with @manatarms  terminology) though, you’d want to find out where the existing `background: #000` and `opacity: .7` are coming from.

Comment (jwineman) [inline packages/cf-style-const/src/variables.js]: As a general rule we try to keep colors in variables so we can track all the different colors we use across the app and eventually standardize them.  I think we should put this `rgba` in [`variables.js`](https://github.com/cloudflare/cf-ui/blob/master/packages/cf-style-const/src/variables.js)

**Comment by prestonp** [comment]
@jwineman Renamed the rgba color in variables.js

File: packages/cf-style-const/src/variables.js (hunk 1 above)
Comment (adamschwartz) [inline packages/cf-style-const/src/variables.js]: That makes sense to me. Then my recommendation would just be the inverse (that those other modal styles should instead use this new variable). Cuz, DRY.

**Comment by jwineman** [comment]
I'm an idiot, I thought you were editing modal, not this package.  My bad.

**Pull request merged**